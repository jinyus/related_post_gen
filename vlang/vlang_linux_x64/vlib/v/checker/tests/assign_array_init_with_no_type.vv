fn main() {
	mut x := []
	println(x)
}
