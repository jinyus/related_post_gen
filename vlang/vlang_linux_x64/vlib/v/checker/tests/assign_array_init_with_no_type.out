vlib/v/checker/tests/assign_array_init_with_no_type.vv:2:11: error: array_init: no type specified (maybe: `[]Type{}` instead of `[]`)
    1 | fn main() {
    2 |     mut x := []
      |              ~~
    3 |     println(x)
    4 | }
vlib/v/checker/tests/assign_array_init_with_no_type.vv:3:2: error: `println` can not print void expressions
    1 | fn main() {
    2 |     mut x := []
    3 |     println(x)
      |     ~~~~~~~~~~
    4 | }
